.gitignore

```diff
@@ -1,4 +1,6 @@
 tags
+go.work
```

Comment:
What's the point of this addition?

Reply:
It's to allow the use of [workspaces](https://go.dev/doc/tutorial/workspaces) during development without them showing up as a changed file.